client/packages/system/src/Report/api/operations.graphql

```diff
@@ -13,7 +13,7 @@ fragment ReportRow on ReportNode {
 
 query reports(
   $storeId: String!
-  $key: String!
+  $key: ReportSortFieldInput!
```

Comment:
Were these added because of warnings? Looks like an attempt to line the types up more accurately, but then in the api.ts file we're doing `key: sortBy.key as ReportSortFieldInput`, which defeats the point?

Looks like that's done thorugh the whole code base, so should probably be a separate refactor issue, but we should have an explicit mapping between the row fields and the sort input... I think I've seen errors something like `sort.0.key is not of type ReportSortFieldInput` before, probably because the compiler isn't warning us here?

Reply:
The version bump has resulted in the errors being reported, where they weren't before. Which means that the pre commit hook will fail without these corrections.

and yep. spot on - we do need a refactor issue to fix the usage! using `as` like this does defeat the purpose of type checking!